server/views/macros/adaTable.njk

```diff
@@ -18,21 +19,26 @@ showTotalPerDateCharged: optional,  defaults to false, should the total be shown
     {% endif %}
 
     {% for ada in params.adas %}
-      <div class="govuk-!-margin-bottom-6">
-        {% if params.checkboxes %}
-          {% set isoDate = ada.dateChargeProved | date("YYYY-MM-DD") %}
-          <div class="govuk-form-group">
-            <div class="govuk-checkboxes govuk-checkboxes--small" data-module="govuk-checkboxes">
-              <div class="govuk-checkboxes__item">
-                <input class="govuk-checkboxes__input row-checkbox" id="prospective-{{ isoDate }}" name="prospective" type="checkbox"value="{{ isoDate }}">
-                <label class="govuk-label govuk-checkboxes__label" for="prospective-{{ isoDate }}">Charge proved {{ ada.dateChargeProved | date("DD MMM YYYY") }}</label>
+      {% if params.cardStyle %}
+        <div class="govuk-summary-card govuk-!-margin-bottom-4">
+          <div class="govuk-summary-card__content equal-padding">
+            <h2 class="govuk-heading-m">Date charge proved {{ ada.dateChargeProved | date("DD MMM YYYY") }}</h2>
+      {% else %}
+        <div class="govuk-!-margin-bottom-6">
+          {% if params.checkboxes %}
```

Comment:
So I don't think there is any page that has checkboxes = true and cardStyle = false.

Reply:
Didn't know the checkboxes were only used by PADAs. I've refactored this